src/scripts/release.sh

```diff
@@ -23,20 +23,20 @@ fi
 
 ORIGIN_URL=$(git remote get-url origin)
 UPSTREAM="anz-bank"
-UPSTREAM_URL=$(echo "$ORIGIN_URL" | sed -n "s/\\(.*\\/\\/github.com\\/\\)\\(..*\\)\\(\\/..*.git\\)/\\1$UPSTREAM\\3/p")
+UPSTREAM_URL=$(echo "$ORIGIN_URL" | sed -E 's|(.*//github.com/)(.+)(/.+\.git)|\1'"$UPSTREAM"'\3|')
```

Comment:
Needs an explanatory comment

Reply:
? This is a merged PR, I've implemented the changes Cam suggested on chat. Do you want me to write that here? Functionality doesn't change, just less bleeding eyes for @camh-anz as he put it.